tests/integration/test_auth.py

```diff
@@ -22,11 +22,70 @@ def test_user_registration(client):
     response = client.json.post(url, json.dumps(credentials))
     assert response.status_code == 201
     expected_keys = [
-        'id', 'email', 'first_name', 'last_name', 'auth_token'
+        'id', 'first_name', 'last_name', 'email', 'gender', 'tshirt_size', 'ticket_id', 'phone_number',
+        'is_core_organizer', 'is_volunteer', 'date_joined', 'is_active', 'is_staff', 'is_superuser', 'auth_token'
     ]
     assert set(expected_keys).issubset(response.data.keys())
 
 
+def test_user_registration_without_required_credentials(client):
+    url = reverse('auth-register')
+    credentials = {
+        'email': 'test@test.com',
+    }
+    response = client.json.post(url, json.dumps(credentials))
+    assert response.status_code == 400
+    expected_keys = [
+        'errors', 'error_type'
+    ]
+    assert set(expected_keys).issubset(response.data.keys())
+    assert response.data['error_type'] == 'ValidationError'
```

Comment:
Where are you checking the name of the corresponding field for which error is coming?